cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -196,15 +197,17 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return frontend.expressionHandler.handle(node.value)
     }
 
+    /**
+     * Translates a Python [AnnAssign](https://docs.python.org/3/library/ast.html#ast.AnnAssign]
+     * into a [Statement].
+     */
     private fun handleAnnAssign(node: Python.AST.AnnAssign): Statement {
-        // TODO: annotations
         val lhs = frontend.expressionHandler.handle(node.target)
+        lhs.type = frontend.typeOf(node.annotation)
```

Comment:
I think that is not necessary, since this should propagate through the assignment expression automatically. Can you see what happens if you remove it?

Reply:
I think this is a bigger discussion to have. I asked @lshala to implement it this way. Think of the following code: `a: int = "some string"` What do we do?
You are suggesting to drop the `int` by taking the type of the rhs.
I disagree. The annotation is there and should be reflected in the graph. We probably want to raise a warning?

This is the AST for the above snippet:
```python
AnnAssign(
      target=Name(id='a', ctx=Store()),
      annotation=Name(id='int', ctx=Load()),
      value=Constant(value='some string'),
      simple=1)]
```

Reply:
Ah. I thought the annotation was on the right side. My bad

Reply:
There are two types / styles of annotations. See the test code example in this PR. One is handled as a `AnnAssign`, while the other adds a `type_comment`.